src/components/rangeslider/attributes.js

```diff
@@ -0,0 +1,47 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+
+module.exports = {
+    visible: {
+        valType: 'boolean',
+        dflt: false,
+        description: 'Determines whether or not the range slider will be visible.'
+    },
+    bordercolor: {
+        valType: 'color',
+        dflt: 'transparent',
+        role: 'style',
+        description: 'Sets the border color of the range slider.'
+    },
+    borderwidth: {
+        valType: 'integer',
+        dflt: 1,
+        role: 'style',
+        description: 'Sets the border color of the range slider.'
+    },
+    backgroundcolor: {
+        valType: 'color',
+        dflt: '#ffffff',
+        role: 'style',
+        description: 'Sets the background color of the range slider.'
+    },
+    height: {
```

Comment:
`height` makes me think of a value in pixels. 

We're using `len` and `thickness` to denote the colorbar's plot area fraction. I'd vote for using `thickness` here, for consistency. 

@chriddyp @cldougl @cpsievert thoughts?